dts/bindings/disk/zephyr,ram-disk.yaml

```diff
@@ -31,5 +31,16 @@ properties:
     type: phandle
     description: |
       Optional phandle to the memory region to be used as a RAM disk,
-      if not used a local buffer is defined for each disk instance.
-      Use it with caution as it makes memory contents easily accessible.
+      if not used, and 'disk-image' property is also not used, a local buffer is
+      defined for each disk instance. Use it with caution as it makes memory
+      contents easily accessible.
```

Comment:
Stray changes.

Reply:
Not a stray change - if the new property gets accepted, the existing description of 'ram-region' property should be updated.

The name of the property, however, has been updated since the original commit to `zephyr,disk-image-path` so I'll update it here, too.